lib/DataHub.js

```diff
@@ -31,55 +31,55 @@ var openFeed = function open(exchange, orderbook) {
 // TODO: Change maps to javascript objects/combine into one Map/Object
 var orderbook = function orderbook(pair) {
   return {
-    pair : pair,
-    last30Day : [],
-    currTopOrderBook : [],
-    OrderBook : {},
-    highbids : new Map(),
-    lowasks : new Map()
+    pair:             pair,
+    last30Day:        [],
+    currTopOrderBook: [],
+    OrderBook:        {},
+    highbids:         new Map(),
+    lowasks:          new Map()
   }
 }
 
 // Overall Exchanges variable constructor
 var createExchanges = function createExchanges (exchangeNames, pairs) {
-  var Exchs = {};
+  var Exchanges = {};
   // Accounting for "all" function call
   if (exchangeNames == 'all') {
     exchangeNames = ["poloniex", "gdax", "kraken", "bitfinex"];
   }
 
   // For each exchange required, creating a Exchange object
-  for (var i = 0; i < exchangeNames.length ; i++) {
+  for (var i = 0; i < exchangeNames.length; i++) {
     if (info[exchangeNames[i] ]) {
-        Exchs[exchangeNames[i] ] = addExchange(exchangeNames[i], pairs);
+        Exchanges[exchangeNames[i] ] = addExchange(exchangeNames[i], pairs);
     }
   }
 
   // Function to open a feed for every exchange/pair needed, loops through every
   //    available key pairs and opens the websocket for that key pair
-  Exchs.open = function () {
-    for (var key in Exchs) {
-      if (Exchs.hasOwnProperty(key)) {
-        for (var key_2 in Exchs[key]){
-          if (Exchs[key].hasOwnProperty(key_2)) {
-            openFeed(key, Exchs[key][key_2]);
+  Exchanges.open = function () {
+    for (var exchName in Exchanges) {
+      if (Exchanges.hasOwnProperty(exchName)) {
+        for (var currencyPair in Exchanges[exchName]) {
+          if (Exchanges[exchName].hasOwnProperty(currencyPair)) {
+            openFeed(exchName, Exchanges[exchName][currencyPair]);
```

Comment:
Nah this is ugly. Can you see if you can use `map()`? If you do, it will look baller. Functional programming ftw!